src/check_systemd_units.py

```diff
@@ -200,11 +208,12 @@ def _check_service(self, timer=False):
         else:
             if self.unit_properties.ActiveState != 'active':
                 return (
-                    self._warn_level, 'the service is inactive'
+                    self._crit_level, 'the service is inactive'
                 )
             if self.unit_properties.SubState == 'exited':
+                # Non one-shot services should not exit
```

Comment:
Above it is called `oneshot` in one word. Let's stick to one version.
```suggestion
                # Non oneshot services should not exit
```